### DISCUSSION TITLE
请问我想将数学字体从cm改为xits，但是报错

### DISCUSSION THREAD

**Discussion by Jamie-obj** [note286/xduts#140] [Q&A] [answered]
![image](https://github.com/note286/xduts/assets/65389500/1dbb19ca-53a2-46cb-bed2-45b480ac969d)
我已将\bm 改为 \symbfit，并将bm包去掉

**Comment by sikouhjw** [comment]
你改了什么？

**Reply by Jamie-obj** [reply]
> 你改了什么？

只改了math-font = xits, 还有\bm 改为 \symbfit

**Reply by Jamie-obj** [reply]
> 你改了什么？

请问文档里的这一项要改吗
![image](https://github.com/note286/xduts/assets/65389500/6f9ff031-3594-47ed-8351-45fe0772e25a)

**Reply by Jamie-obj** [reply, marked as answer]
我的问题，，，符号对照表里的命令没改。。。

**Comment by Ziya-miao** [comment]
论文中数学字体不对呀，文档中需要修改的吧

**Reply by sikouhjw** [reply]
你在说什么？

**Comment by Ziya-miao** [comment]
数学公式字体，是xits吗？感觉不太对的样子

**Reply by sikouhjw** [reply]
https://github.com/note286/xduts/blob/3237abfaf40e7c94a8e2b200bd4f5b2cb9b2c8bf/xduts.dtx#L582-L586
请您阅读手册后再使用模板
